server/main.py

```diff
@@ -47,33 +43,15 @@ def get_application() -> FastAPI:
 
 app = get_application()
 
-# TODO: move to API folder
-@app.get("/api/v1/coastlines")
-async def get_coastlines(
-    db_session: AsyncSession = Depends(get_db),
-) -> models.Coastline:
-
-    # `selectinload`: alternative approach to `joinedload`
-    stmt = (
-        select(models.Coastline)
-        .options(joinedload(models.Coastline.buoy, innerjoin=True))
-        .order_by(models.Coastline.id)
-    )
-    response = await db_session.execute(stmt)
-
-    """TODO
-    * Add error handling
-    """
-
-    return [schemas.Coastline.from_orm(row.Coastline).dict() for row in response]
-
 
 @app.get("/api/v1")
 def root():
-    return {"message": "Hello world!"}
+    return {"message": "Welcome to Version 1, Senpai UwU"}
 
 
 # TODO: add routes
+# Coastlines Route
+app.include_router(coastlines.router)
```

Comment:
What do you think about the pattern proposed by the creator of FastAPI in [his template repo](https://github.com/tiangolo/full-stack-fastapi-postgresql/tree/master)?

- specifically, this [line](https://github.com/tiangolo/full-stack-fastapi-postgresql/blob/master/%7B%7Bcookiecutter.project_slug%7D%7D/backend/app/app/main.py#L21) in main.py
- and the way routes are set [here](https://github.com/tiangolo/full-stack-fastapi-postgresql/blob/master/%7B%7Bcookiecutter.project_slug%7D%7D/backend/app/app/api/api_v1/api.py).
- and how every [endpoint is a file nested under the version number](https://github.com/tiangolo/full-stack-fastapi-postgresql/tree/master/%7B%7Bcookiecutter.project_slug%7D%7D/backend/app/app/api/api_v1/endpoints) instead of an additional folder.

What's the use case for your proposed file structure?

There's no wrong answer. The beauty is we can always refactor in the future and adapt! I'd like to better understand your thought process. :)

Reply:
I don't really understand why he set his api router the way that he did. For the time being it seems unnecessary to me. But I do like the `server/api/v1/endpoints/*` file structure. I changed it to that as it make things more organized and readable. Makes more sense to have just a single endpoints directory rather than multiple directories each having the same name as the route names. eg coastlines/coastlines.py seems redundant.

Reply:
Sounds good. Let's start there!